src/Symfony/Bridge/Doctrine/Validator/Constraints/UniqueEntityValidator.php

```diff
@@ -167,13 +167,19 @@ public function validate($entity, Constraint $constraint)
         $errorPath = null !== $constraint->errorPath ? $constraint->errorPath : $fields[0];
         $invalidValue = isset($criteria[$errorPath]) ? $criteria[$errorPath] : $criteria[$fields[0]];
 
-        $this->context->buildViolation($constraint->message)
+        $constraintViolationBuilder = $this->context->buildViolation($constraint->message)
             ->atPath($errorPath)
             ->setParameter('{{ value }}', $this->formatWithIdentifiers($em, $class, $invalidValue))
             ->setInvalidValue($invalidValue)
             ->setCode(UniqueEntity::NOT_UNIQUE_ERROR)
             ->setCause($result)
-            ->addViolation();
+        ;
+        foreach ($fields as $field) {
+            $invalidValue = ($criteria[$field]) ? $this->formatWithIdentifiers($em, $class, $criteria[$field]) : '';
```

Comment:
the `$invalidValue` variable were previously assigned because reused in `setInvalidValue($invalidValue)`
here, you don' need to use this variables, you can inline the code in `setParameter`. (or used another variable name).

Reply:
@jderusse thank you for reviewing the code. I just fixed it.